core/coreunix/add.go

```diff
@@ -114,6 +115,14 @@ func (adder *Adder) SetMfsRoot(r *mfs.Root) {
 
 // Constructs a node from reader's data, and adds it. Doesn't pin.
 func (adder *Adder) add(reader io.Reader) (ipld.Node, error) {
+	if adder.FileWrapper != nil {
+		wr, err := adder.FileWrapper(reader)
```

Comment:
So, this is the single point of entry of a reader to the chunker right? (The other similar calls I'm finding in the code are for testing purposes only.)